js/components/operatorSignIn/attestation.tsx

```diff
@@ -158,7 +159,7 @@ export const SignaturePrompt = ({
           inputMode="numeric"
           defaultValue={defaultValue}
           onChange={(evt) => {
-            onChange(evt.target.value);
+            onChange(removeLeadingZero(evt.target.value));
```

Comment:
Hmm, doing it this way means the react state is different from what's in the input. I think it'd work, but it seems like it has potential to end up with weird bugs if we change anything, like the input not visually responding if you type a 0.

An alternative would be to include the text as typed, including leading 0s, in the react state. And then remove the leading zeroes wherever we use that variable.

Reply:
Point taken 👍 will fix

Reply:
discussed in slack- adding comments instead of code change to prevent `removeLeadingZero` from having to make it to the modal as well.